apps/expo/tsconfig.json

```diff
@@ -1,6 +1,10 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "~/*": ["../../apps/expo/src/*"]
```

Comment:
Why ../../app/expo ?

I know i had issues getting this to work when i added the nextjs aliases. Have you tried making a build after adding these aliases?

Reply:
@juliusmarminge, you are right twice, thanks for pointing those out to me. I don't know what was in my mind.

1. There's no need to go back and forward with the path of course... I guess something bad happened with my types resolution and when it worked I stopped reasoning.
2. I forgot to configure the bundler alias. Now this PR has 1 new commit which configures it and I successfully built the expo app with this new config.